CHANGELOG.md

```diff
@@ -6,8 +6,8 @@
 - [#2572](https://github.com/scality/metalk8s/issues/2572) - Bump CoreDNS
 version to 1.6.2 (PR [#2575](https://github.com/scality/metalk8s/pull/2575))
 
-- [#2674](https://github.com/scality/metalk8s/issues/2674) - Bump K8S version
```

Comment:
Let's keep the issue reference, since it mentions some fixed CVEs.

Reply:
:ok: